test/fixtures/receipt_mailer/receipt_email

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <style>
+      /* Email styles need to be inline */
+    </style>
+  </head>
+
+  <body>
+    <p>Hello,</p>
+
+<p>You are receiving this message to confirm the successful submission of the
+electronic version of your July 2017 thesis "MyString" to the MIT Libraries.
```

Comment:
Do we have a requirement that graduation date can't be too far in the past? Will this test eventually fail on that basis?

Reply:
Hmmm. Good question. The Thesis fixture file that is loaded to compare against this email fixture would become invalid in that case... so this test isn't the problem so much as the upstream fixture itself. I'll open a ticket to review that concern as I think it is valid but outside the scope of the email test itself.

Reply:
https://mitlibraries.atlassian.net/browse/TI-66